front/src/Footer.js

```diff
@@ -11,6 +11,8 @@ class Header extends Component {
           L'API&nbsp;<a href="https://www.ssllabs.com/">Qualys SSL Labs</a>&nbsp;est utilisée pour calculer les notes SSL
         </p>
         <p>Dernière mise à jour le {this.props.lastUpdated}</p>
+
+        <img src="http://stats.data.gouv.fr/piwik.php?idsite=25&rec=1" alt="" />
```

Comment:
Je pense savoir le faire facilement avec les scripts. Je jete un œil demain.

Thibaut

Le mardi 23 août 2016, Matti Schneider <notifications@github.com> a écrit :

> In front/src/Footer.js
> <https://github.com/sgmap/verif.site/pull/18#discussion_r75861693>:
>
> > @@ -11,6 +11,8 @@ class Header extends Component {
> >            L'API&nbsp;<a href="https://www.ssllabs.com/">Qualys SSL Labs</a>&nbsp;est utilisée pour calculer les notes SSL
> >          </p>
> >          <p>Dernière mise à jour le {this.props.lastUpdated}</p>
> > +
> > +        <img src="http://stats.data.gouv.fr/piwik.php?idsite=25&rec=1" alt="" />
>
> Oh. Ok. Then indeed go with it, but we’re going to miss a huge lot of
> information such as screen size, reliable browser identification…
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/sgmap/verif.site/pull/18/files/4b35c1b642bb6fb5e05b98a73875b52f06c52779#r75861693>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFlwwcJvsHe6mzOmY-D4OIQS0pHYJ6iwks5qivGbgaJpZM4JqxG2>
> .
>


-- 
Thibaut Géry